### ISSUE TITLE
Internal Server Error 

### ISSUE THREAD

**Issue by Kaseax** [very-doge-wow/amazify#27]
Hey! Just wanted to give the app a try. Everything works, but if I click on the last button, I'll get a "Internal Server Error"

![image](https://github.com/user-attachments/assets/3ce397ac-bd58-4ef7-b2b8-67241a9cb8a3)

cc @very-doge-wow

**Comment by very-doge-wow** [comment]
Hi, the error message indicates that the amazon user object couldn't be retrieved. Have you set up the Amazon OAuth as described in the [Readme](https://github.com/very-doge-wow/amazify?tab=readme-ov-file#-amazon-authentication-workaround)?

I also haven't tested the app with Python 3.9, I was using 3.11.

**Comment by Kaseax** [comment]
Yes, I did just like that. Clicked on "Get Token", logged in and copied it.

**Comment by very-doge-wow** [comment]
Just tried it myself and it indeed seems to be broken... This is the response by the amazon API:

```
{"status":401,"code":"INVALID_CLIENT_ID","message":"Unable to identify client. Check the x-api-key header.","reference":"4d8efdc0-22ed-4a24-87d0-839a9b048495"}
```

It may very well be that the workaround using the generated token by the interactive API reference has been patched and doesn't work anymore 🙁 

I'll have to do some more debugging when I get to it...

**Comment by Kaseax** [comment]
Okay, that's sad. So there's no way to use your app at the moment anymore?

**Comment by very-doge-wow** [comment]
Actually I believe it's a bug in the interactive API reference, which seems to issue invalid x-api-key headers.
I think that's the reason because even when using it directly, it returns the same message:

![image](https://github.com/user-attachments/assets/4f17a214-b23e-441b-9cd2-ed01d09d2976)

So yes, the app can't be used at the moment unless amazon fix the API reference.

**Comment by very-doge-wow** [comment]
Sadly the Amazon Music API is still in beta stage and no official support is available. I have however created a new thread in the Amazon Developer Forum. I would not however get my hopes up, as activity seems to be quite low there...

**Comment by very-doge-wow** [comment]
Unexpected answer from Amazon:

The dashboard isn't supposed to be publicly available as the API is still in beta stage. So the fact we were able to use it was rather by mistake and I don't think they'll employ any effort to fix it for us. 😞
